app/templates/components/nypr-brick-item.hbs

```diff
@@ -0,0 +1,23 @@
+<li class="{{item.attributes.template}} item--trending"
+    style="background-image: url({{item.attributes.imageMain.url}})">
```

Comment:
any value set to the `style` attribute needs to be escaped using `Ember.String.htmlSafe` or else you'll get warnings in the console.